webview.go

```diff
@@ -54,16 +54,20 @@ type webview struct {
 }
 
 // New creates a new webview in a new window.
-func New(debug bool) WebView { return NewWindow(debug, nil) }
+func New(debug bool) WebView { return NewWindow(debug, "", nil) }
+
+// NewWithDataPath creates a new webview in a new window.
+func NewWithDataPath(debug bool, dataPath string) WebView { return NewWindow(debug, dataPath, nil) }
 
 // NewWindow creates a new webview using an existing window.
-func NewWindow(debug bool, window unsafe.Pointer) WebView {
+func NewWindow(debug bool, dataPath string, window unsafe.Pointer) WebView {
```

Comment:
This appears to be an API break on NewWindow. Probably need to make a new function.